src/runtime_src/core/edge/user/aie/aie.h

```diff
@@ -125,6 +125,48 @@ class Aie {
     void
```

Comment:
Copyright needs to be updated (line 3).  Being that we are part of AMD now, the following addition should be made.
`Copyright (C) Advanced Micro Devices, Inc.`